### ISSUE TITLE
Model Convergence Indicator

### ISSUE THREAD

**Issue by already-taken-m17** [orenmel/context2vec#1]
Hi,
During the training, how the number of epochs are getting decided ?
As I see, no part of training data has been reserved for evaluation during training of model. So, how can I be assured that model has converged during training ?

**Comment by orenmel** [comment]
Correct. As in word2vec, there is no such validation set built into the training process, but indeed it would probably be useful to have this (as done when training language models, etc.). I guess that if the train set is large enough then you can look at the train error without too much of a risk of over-fitting.
